### PR TITLE
John/docs

### PR DESCRIPTION
**Pull request by Mac-Genius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,174 +8,23 @@ programming as well as a community of excited developers.
...
-SECRET_KEY = 'developement key'
-DATA_FOLDER = '/Path/to/your/clone/auacm/auacm/app/data'
-TEMP_DIRECTORY = '/Path/to/your/clone/auacm/auacm/temp'
+1. cd to the `/../auacm/setup/` directory
 
-TEST_USERNAME = 'will'
-TEST_PASSWORD = 'password'
-```
-Your `config.py` file belongs in the `auacm/auacm/app` directory.
+2. execute the script `./macos_setup.py`. This will walk you through setting up the development environment for the site.
```
Comment (mcprice30) [inline README.md]: Is this a shell script or a python file? This and the installation instructions disagree.

File: README.md [hunk 2]
```diff
@@ -8,174 +8,23 @@ programming as well as a community of excited developers.
...
-   [Docker Compose](https://docs.docker.com/compose/install/).
-   * Docker is a *Linux containerization engine*. Containers are like virtual
-     machines, but lower level and much more lightweight. The Docker engine is
-     what creates, manages, and deletes the individual containers.
+1. cd to the `/../auacm/setup/` directory
 
-2. **Mac and Windows users** need to install a Docker machine to run docker
-   containers on (this requires VirtualBox as well). For OSX, create a Docker
-   machine with
+2. execute the script `./ubuntu_setup.py`. This will walk you through setting up the development environment for the site.
```
Comment (mcprice30) [inline README.md]: Ditto

**Review by mcprice30 — COMMENTED** [review]
Other than the script names, looks good.

File: README.md (hunk 1 above)
Comment (Mac-Genius) [reply README.md]: Oops, should be a shell script.

**Review by BrandonLMorris — APPROVED** [review]

**Pull request merged**